src/eru.Infrastructure/eru.Infrastructure.csproj

```diff
@@ -9,8 +9,14 @@
     </ItemGroup>
 
     <ItemGroup>
-      <PackageReference Include="AutoMapper" Version="10.0.0" />
       <PackageReference Include="FluentAssertions" Version="5.10.3" />
+      <PackageReference Include="Hangfire" Version="1.7.12" />
+      <PackageReference Include="Hangfire.Dashboard.BasicAuthorization" Version="1.0.2" />
+      <PackageReference Include="Hangfire.MemoryStorage" Version="1.7.0" />
+      <PackageReference Include="Microsoft.EntityFrameworkCore.InMemory" Version="3.1.6" />
+      <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="3.1.6" />
+      <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="3.1.6" />
+      <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="3.1.6" />
     </ItemGroup>
 
     <ItemGroup>
```

Comment:
Delete the whole <ItemGroup> containing platform specific reference to Microsoft.Extensions.DependencyInjection.Abstractions package (which is anyway available in app as MediatR dependency)